### ISSUE TITLE
Noto Sans Javanese does not support four above-base marks on one base

### ISSUE THREAD

**Issue by dscorbett** [googlefonts/noto-fonts#2340]
### Font 
NotoSansJavanese-Regular.otf

### Where the font came from, and when
Site: https://github.com/googlefonts/noto-fonts/blob/3c542c9d670e518131c73c9f9f4a425ce3857aeb/unhinted/otf/NotoSansJavanese/NotoSansJavanese-Regular.otf
Date: 2022-04-25

### Font version
Version 2.003

### Issue
This font supports combinations of up to three above-base marks, but not four above-base marks. The fourth mark is currently mispositioned relative to the rest and overlaps them. For example, as it supports <cecak telu, wulu, panyangga> and <wulu, panyangga, layar>, it should support the combined sequence <cecak telu, wulu, panyangga, layar>.

### Character data
ꦥ꦳ꦶꦀꦂ
U+A9A5 JAVANESE LETTER PA
U+A9B3 JAVANESE SIGN CECAK TELU
U+A9B6 JAVANESE VOWEL SIGN WULU
U+A980 JAVANESE SIGN PANYANGGA
U+A982 JAVANESE SIGN LAYAR

### Screenshot
![ꦥ꦳ꦶꦀꦂ](https://user-images.githubusercontent.com/1124347/165197319-c4f7c3d6-e4f7-4e73-a950-341e446f78e6.png)

**Comment by r12a** [comment]
@dscorbett do you have examples of U+A980: JAVANESE SIGN PANYANGGA​ being used other than for OM ?

**Comment by dscorbett** [comment]
No, but the font does go out of its way to support U+A980 in other contexts. https://id.wikipedia.org/wiki/Aksara_Jawa says “umumnya hanya ditemukan dalam salinan lontar Bali untuk menuliskan kata keramat seperti <i>[ong](https://id.wikipedia.org/wiki/Om)</i> ꦎꦀ”, implying that it is used in other syllables.

**Comment by adtbayuperdana** [comment]
I think I can give some context to this. Like @r12a said, panyangga is normally used only for holy syllables such as OM in Hindu or Buddhist texts. U+A9B3 JAVANESE SIGN CECAK TELU is a nukta-like character that was originally developed to accommodate Arabic loanwords in Islamic texts, when Hindu-Buddhist influence has already waned in Java. So, in native texts, panyangga and cecak telu occurred in texts with very different theme and it is not likely that both could occur in the same letter. Therefore,  panyangga + wulu, layar, cecak etc combinations might be possible considering that they are all part of the original indic repertoire, but combination with panyangga + cecak telu are even more unlikely to occur. At this juncture, the combination is hypothetically possible but not attested anywhere as far as I know.
